TestFoundation/TestNSString.swift

```diff
@@ -228,6 +229,18 @@ class TestNSString : XCTestCase {
         let string = NSString(CString: bytes.map { Int8(bitPattern: $0) }, encoding: NSUTF8StringEncoding)
         XCTAssertNil(string)
     }
+    
+    func test_FromContentOfFile() {
+        let testFilePath = testBundle().pathForResource("NSStringTestData", ofType: "txt")
+        
+        do {
+            let str = try NSString(contentsOfFile: testFilePath, encoding: NSUTF8StringEncoding)
```

Comment:
@phausler  thank you,I fix it at ccommit of 7c9d732. I cant build the test because of the eroor mentioned before ,even I merge the newest code to my fork branch . the error is in the 246 line of NSHTTPCookie.swift, error message is
  
     /swift-corelibs-foundation/Foundation/NSHTTPCookie.swift:246:13: 
    All stored properties of a class instance must be initialized before returning nil from an initializer